src/CSSList/CSSBlockList.php

```diff
@@ -96,42 +96,42 @@ protected function allValues(
      */
     protected function allSelectors(array &$result, $specificitySearch = null): void
     {
-        /** @var array<int, DeclarationBlock> $aDeclarationBlocks */
-        $aDeclarationBlocks = [];
-        $this->allDeclarationBlocks($aDeclarationBlocks);
-        foreach ($aDeclarationBlocks as $oBlock) {
+        /** @var array<int, DeclarationBlock> $declarationBlocks */
+        $declarationBlocks = [];
+        $this->allDeclarationBlocks($declarationBlocks);
+        foreach ($declarationBlocks as $oBlock) {
             foreach ($oBlock->getSelectors() as $selector) {
                 if ($specificitySearch === null) {
                     $result[] = $selector;
                 } else {
-                    $sComparator = '===';
-                    $aSpecificitySearch = \explode(' ', $specificitySearch);
-                    $iTargetSpecificity = $aSpecificitySearch[0];
-                    if (\count($aSpecificitySearch) > 1) {
-                        $sComparator = $aSpecificitySearch[0];
-                        $iTargetSpecificity = $aSpecificitySearch[1];
+                    $comparator = '===';
+                    $specificitySearchResults = \explode(' ', $specificitySearch);
+                    $targetSpecificity = $specificitySearchResults[0];
+                    if (\count($specificitySearchResults) > 1) {
+                        $comparator = $specificitySearchResults[0];
+                        $targetSpecificity = $specificitySearchResults[1];
                     }
-                    $iTargetSpecificity = (int) $iTargetSpecificity;
-                    $iSelectorSpecificity = $selector->getSpecificity();
-                    $bMatches = false;
-                    switch ($sComparator) {
+                    $targetSpecificity = (int) $targetSpecificity;
+                    $selectorSpecificity = $selector->getSpecificity();
+                    $hasMatches = false;
```

Comment:
Apologies, I missed this in the previous review.

I don't think "has" is correct here.  The equivalent linguistic phrase is "it matches" rather than "it has matches"...

Reply:
Better late than never. ;-) I've come up with a different name and repushed.

Reply:
Maybe `comparatorMatched`?